ejb3/src/main/java/org/jboss/as/ejb3/deployment/processors/EjbJndiBindingsDeploymentUnitProcessor.java

```diff
@@ -161,8 +165,12 @@ private void setupJNDIBindings(EJBComponentDescription sessionBean, DeploymentUn
             }
 
             // log EJB's ejb:/ namespace binding
-            final String ejbNamespaceBindingName = sessionBean.isStateful() ? ejbNamespaceBindingBaseName + "!" + viewClassName + "?stateful" : ejbNamespaceBindingBaseName + "!" + viewClassName;
-            logBinding(jndiBindingsLogMessage, ejbNamespaceBindingName);
+            final String ejbNamespaceBindingName =  isEjbNamespaceBindingBaseName == true ? "": sessionBean.isStateful() ? ejbNamespaceBindingBaseName + "!" + viewClassName + "?stateful" : ejbNamespaceBindingBaseName + "!" + viewClassName;
```

Comment:
Maybe it would be a bit more elegant to evaluate this expression in the if below and thus avoid artificial empty string initialization.

Reply:
Hi @tadamski ,
Do you think below code change is fine. If ok ill make changes and resend pull request. 

 final String ejbNamespaceBindingName =  sessionBean.isStateful() ? ejbNamespaceBindingBaseName + "!" + viewClassName + "?stateful" : ejbNamespaceBindingBaseName + "!" + viewClassName;

if(!isEjbNamespaceBindingBaseName){
                logBinding(jndiBindingsLogMessage, ejbNamespaceBindingName);
}

Reply:
@tadamski could you please review :)

Reply:
@moulalis yes, this looks fine

Reply:
Thanks @tadamski . I have resent PR for this change.